truss/templates/server/model_wrapper.py

```diff
@@ -419,7 +456,20 @@ def _load_impl(self):
 
         self._maybe_model_descriptor = ModelDescriptor.from_model(self._model)
 
-        if hasattr(self._model, "load"):
+        asyncio.create_task(self.poll_for_environment_updates())
```

Comment:
could you save the task as a property on the class? Something like:

```
self.poll_for_environment_updates_task = asyncio.create_task(...)
```

The risk with how this is written now is that the task immediately goes out of scope & could get garbage collected